### PR TITLE
Gpavlov/setup cypress

### PR DESCRIPTION
**Pull request by gpavlov2016**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ deployed on the Flow Testnet.
 
 _🛠 This project requires `NodeJS v16.x` or above._ See: [Node installation instructions](https://nodejs.org/en/) <br/>
 _🛠 This project requires `flow-cli v0.32.1` or above._ See: [Flow CLI installation instructions](https://docs.onflow.org/flow-cli)
+_🛠 This project requires `Python v3.6` or above._ See: [Python installation instructions](https://www.python.org/downloads)
```
Comment (10thfloor) [inline README.md:25]: Should we have Python as a "hard" requirement, or can we annotate this to make it clear this is only necessary if you want to run the integration tests yourself, locally? 

Installing Python is not always straightforward, and we want to keep this project as accessible as possible.

File: .github/workflows/integration-local.yml [hunk 2]
```diff
@@ -44,3 +45,70 @@ jobs:
...
+
+        - name: Cypress run
+          uses: cypress-io/github-action@v4
+          with:
+            start: npm run dev:emulator
+            wait-on: http://localhost:3001
+
+        # - name: Print logs
+        #   if: always()
+        #   run: npx pm2 logs
```
Comment (10thfloor) [inline .github/workflows/integration-local.yml:95]: Can we remove the commented sections in this file?

**Review by 10thfloor — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (gpavlov2016) [reply README.md:25]: Python is required on some systems to run npm i when there are no prebuilt binaries for the package you are installing. I ran into this problem with sqlite package on Apple M1.

File: .github/workflows/integration-local.yml (hunk 2 above)
Comment (gpavlov2016) [reply .github/workflows/integration-local.yml:95]: Absolutely, removed and updated PR

File: README.md (hunk 1 above)
Comment (gregsantos) [reply README.md:25]: Just ran into this as well

**Review by 10thfloor — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (10thfloor) [reply README.md:25]: Ah yes. SQLite.

**Pull request merged**